### PR TITLE
fix(RHINENG-8555): Handle empty tag filter values

### PR DESCRIPTION
**Pull request by mhuth**
- Sends namespace/key= if there is no value, instead of null

- [x] The commit message has the Jira ticket linked
- [x] PR has a short description
- [x] Screenshots before and after the change are added
- [x] Tests for the changes have been added
- [ ] README.md is updated if necessary
- [ ] Needs additional dependent work

## Summary by Sourcery

Handle tag filters with empty or missing values by preserving the key and equals sign in the generated tag string.

Bug Fixes:
- Ensure tag filters with empty, null, or undefined values generate tags in the form namespace/key= instead of using null or omitting the equals sign.

Tests:
- Add unit tests covering tag filters with empty string, null, and undefined values to verify the generated tag format.

Screenshots:
Showing empty values are not sent as 'namespace/key=null' but instead as 'namespace/key=':
Screenshot from 2026-06-10 11-48-39
Screenshot from 2026-06-10 11-48-53

The 400 error from the API is fixed in this PR: https://github.com/RedHatInsights/advisor-backend/pull/170

### CODE REVIEW

File: src/PresentationalComponents/Common/Tables.js [hunk 1]
```diff
@@ -37,7 +37,8 @@ export const buildTagFilter = (tagFilters) => {
     ? {
         tags: tagFilters.flatMap((tagFilter) =>
           tagFilter.values.map(
-            (tag) => `${tagFilter.key}/${tag.tagKey}=${tag.value}`,
+            (tag) =>
+              `${tagFilter.key}/${tag.tagKey}=${tag.value ? `${tag.value}` : ''}`,
```
[the remark this answers is not in the document]
Comment (mhuth) [reply src/PresentationalComponents/Common/Tables.js]: Fixed!

**Review by bastilian — APPROVED** [review]
LGTM! Thank you @mhuth!
